src/retinanet.py

```diff
@@ -267,49 +268,46 @@ def __init__(self, aspect_ratios, scale_ratios, num_anchors):
         self.aspect_ratios = aspect_ratios
         self.scale_ratios = scale_ratios
         self.num_anchors = num_anchors
-        self.anchor_wh = self._get_anchor_hw()
-
-    def _get_anchor_hw(self):
-        """Compute anchor height and width for each feature map.
+        self.anchor_wh = self._get_anchor_wh()
 
+    def _get_anchor_wh(self):
+        '''Compute anchor width and height for each feature map.
         Returns:
-          anchor_hw: (tensor) anchor hw, sized [#fm, #anchors_per_cell, 2].
-        """
-        anchor_hw = []
+          anchor_wh: (tensor) anchor wh, sized [#fm, #anchors_per_cell, 2].
+        '''
+        anchor_wh = []
         for s in self.anchor_areas:
             for ar in self.aspect_ratios:  # w/h = ar
-                h = sqrt(s / ar)
+                h = sqrt(s/ar)
                 w = ar * h
                 for sr in self.scale_ratios:  # scale
-                    anchor_h = h * sr
-                    anchor_w = w * sr
-                    anchor_hw.append([anchor_h, anchor_w])
+                    anchor_h = h*sr
+                    anchor_w = w*sr
+                    anchor_wh.append([anchor_w, anchor_h])
         num_fms = len(self.anchor_areas)
-        return torch.Tensor(anchor_hw).view(num_fms, -1, 2)
+        return torch.Tensor(anchor_wh).view(num_fms, -1, 2)
 
     def _get_anchor_boxes(self, input_size):
-        """Compute anchor boxes for each feature map.
-
+        '''Compute anchor boxes for each feature map.
         Args:
-          input_size: (tensor) model input size of (h, w).
-
+          input_size: (tensor) model input size of (w,h).
         Returns:
           boxes: (list) anchor boxes for each feature map. Each of size [#anchors,4],
                         where #anchors = fmw * fmh * #anchors_per_cell
-        """
+        '''
         num_fms = len(self.anchor_areas)
-        fm_sizes = [(input_size / pow(2., i + 3)).ceil() for i in range(num_fms)]  # p3 -> p7 feature map sizes
+        fm_sizes = [(input_size/pow(2.,i+3)).ceil() for i in range(num_fms)]  # p3 -> p7 feature map sizes
```

Comment:
This is not PEP8 right?

Reply:
@jakubczakon I've copied back some lines from retina repo to deal with w, h mess, didn't realize it was like that